releases/Process/Major Release Process.md

```diff
@@ -59,91 +56,108 @@ The release process is managed by sig-release. To request an update to this docu
+1. Follow the steps outlined for the pre-release task. In the pre-release task step (1) Create a feature list file at [https://github.com/o3de/sig-release/tree/main/releases](https://github.com/o3de/sig-release/tree/main/releases)  under the appropriate release, for example  [22100 feature list.md](https://github.com/o3de/sig-release/blob/main/releases/22.10.0/22100%20feature%20list.md "22100 feature list.md") . The initial feature list file that you create should have a heading for each SIG.  
+For pre-release step (4) "Review closed issues to see if anything else should be added."  
+	* You will need to Refer to all O3DE Roadmaps to understand which features have been delivered to the codebase.
+	  * SIG Roadmaps, for example [the O3DE UI/UX Roadmap.](https://github.com/orgs/o3de/projects/9)
+	  * Other roadmaps,  including those published by foundation members. For example [the Roadmap of AWS Contributions to O3DE.](https://github.com/orgs/o3de/projects/11)
+	* For each roadmap item delivered to the o3de codebase since the last release, enter a summary into the feature list.
+	* If you need assistance, comment under the GHI or reach out to the SIG owning that area of the engine
+	* (Optional) Review the submitted pull requests or closed Github Issues. The reason to do this would be to discover any features that were not on the Roadmap, potentially features that were accidentally missed. 
+3. Once you have created the feature list, post a note to #sig-all in discord requesting the SIGs to review and update if needed via Pull Request.
+4. Notify the marketing committee once the feature list is mostly (80%) done (you have some input from each SIG, and they indicate that they "most" of their work submitted). They may ask for a summary of the release and key features. It is up to the release manager to provide this guidance to the marketing committee. The release manager may discuss with the Release SIG or any other SIG to figure this out.
 
-### 1.3 Using Milestones
-//TODO potentially document this. We have not been using milestones consistently, so before doing this we need to speak with QA and Docs to verify usage. 
+### 1.3 More about how Milestones are used
+For each release, Github milestones are used to track bugs found in the stabilization branches of each repo involved in the release. For example, for the 22.20.0 release, the [Release/2210 milestone](https://github.com/o3de/o3de/milestone/11) captures all of the bugs found during release stabilization of the O3DE repo.
 
-Exit Criteria: Ideally, all of the planning tasks are complete on the project board. If not all tasks are complete, it is up to the release manager and co-release manager to understand and accept any risk before moving onto Phase 2. 
+Milestones are used as follows:
+* When bugs are found in the stabilization branch, the reported bug is added to the Milestone. (these instructions are posted in the Release "Key Information")
```

Comment:
Who will be responsible to add the bug to the milestone?

Reply:
the reporter of the bug should be doing this, but this raises a couple of questions - 1) is everyone actually able to do that? (I am not sure if assigning a milestone requires a privilege (@vincent6767  Can you try to add an issue in the O3DE repo to a milestone? (the remove it)? Ideally we need to test with someone from the community   2) Should the reported actually be responsible for doing that, or should the release manager be responsible for it?

Reply:
I noted this in my Jan 3 commit.